apps/ejabberd/src/ejabberd_s2s.hrl

```diff
@@ -0,0 +1,23 @@
+%% These error macros will change the xmpp errors(xml) to binaries
+%% All errors are defined in mongoose_xmpp_errors.erl
+%% Previously they were defined in up to three different modules
+
+-define(INVALID_NS_ERR_TO_BIN,
```

Comment:
Used only twice. We may get rid of the macro and replace it with `exml:to_binary(mongoose_xmpp_errors:invalid_namespace()))` in the code.